frontend/src/metabase/collections/components/CollectionMenu/CollectionMenu.tsx

```diff
@@ -19,26 +20,23 @@ export interface CollectionMenuProps {
   onUpdateCollection: (entity: Collection, values: Partial<Collection>) => void;
 }
 
+const mergeArrays = (arr: any[][]) => {
```

Comment:
```suggestion
const mergeArrays = <T>(arr: T[][]) => {
```
I think we want to make this generic to avoid the return type from turning into `any[]`

Reply:
Making it generic is giving me a hard time, but I can type it to `ReactNode` and it's happy, so I'll do that